script/__snapshots__/migrate-test-e2e.ts.snap

```diff
@@ -214,6 +214,7 @@ exports[`expected file changes > knip.json 1`] = `
  {
  	"$schema": "https://unpkg.com/knip@latest/schema.json",
 -	"entry": ["script/*e2e.js", "src/index.ts!", "src/**/*.test.*"],
+-	"ignoreDependencies": ["all-contributors-cli"],
```

Comment:
I think it was coming in as a nested dev dependency before. Now that I've had to make it explicit in `package.json`, Knip is reporting it. Ah well.